From 3005fc0f7caff5a375fa2e353d3bcbadfd59382e Mon Sep 17 00:00:00 2001
From: Chris Bagwell <chris@cnpbagwell.com>
Date: Tue, 23 Aug 2011 19:31:51 -0500
Subject: [PATCH] reduce 2-finger scroll holdoff time

The only reason not to immediately go into 2 finger scroll
and zoom mode are:

1) Make sure user is not going to do a 2 finger time based gesture
(only 2 finger tap supported).  Waiting only taptime is fine for this.
2) Wait for some movement of fingers to see if its a zoom vs. scroll.
This is movement based and not time based.  Reducing time doesn't seem
to change behavior here.

Signed-off-by: Chris Bagwell <chris@cnpbagwell.com>
---
 src/wcmTouchFilter.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/wcmTouchFilter.c b/src/wcmTouchFilter.c
index 433e12f..72cc937 100644
--- a/src/wcmTouchFilter.c
+++ b/src/wcmTouchFilter.c
@@ -292,7 +292,7 @@ void wcmGestureFilter(WacomDevicePtr priv, int channel)
 	/* process complex two finger gestures */
 	else {
 		CARD32 ms = GetTimeInMillis();
-		int taptime = 2 * common->wcmGestureParameters.wcmTapTime;
+		int taptime = common->wcmGestureParameters.wcmTapTime;
 
 		if (ds[0].proximity && ds[1].proximity &&
 		    (taptime < (ms - ds[0].sample)) &&
-- 
1.7.4.1

